From 169febe60dac5352241392620ac31832469d745d Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Thu, 25 Oct 2018 10:43:20 +0800
Subject: [PATCH] clk: rockchip: rk1808: add pll 100M config parameters

PPLL 100M use refdiv =1 fbdiv = 150, postdiv1= 6,
postdiv2=6, vco= 3.6G, is best for pcie.

Change-Id: I40eb1a71c5025a68cab65ec56d2c2a7725d30c63
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk1808.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk1808.c b/drivers/clk/rockchip/clk-rk1808.c
index 1f3d351a80af..0c242398b501 100644
--- a/drivers/clk/rockchip/clk-rk1808.c
+++ b/drivers/clk/rockchip/clk-rk1808.c
@@ -67,6 +67,7 @@ static struct rockchip_pll_rate_table rk1808_pll_rates[] = {
 	RK3036_PLL_RATE(312000000, 1, 52, 2, 2, 1, 0),
 	RK3036_PLL_RATE(216000000, 1, 72, 4, 2, 1, 0),
 	RK3036_PLL_RATE(200000000, 1, 200, 6, 4, 1, 0),
+	RK3036_PLL_RATE(100000000, 1, 150, 6, 6, 1, 0),
 	RK3036_PLL_RATE(96000000, 1, 64, 4, 4, 1, 0),
 	{ /* sentinel */ },
 };
-- 
2.35.3

